README.md

```diff
@@ -74,6 +461,8 @@ You can take a look at the [Roadmap](https://github.com/chrisvdm/redwoodjs-strip
 - Webhook handling 
 - LoFi Cart
 
+## Thank you
 
+The teams from [Stripe](https://stripe.com/) and [RedwoodJS](https://redwoodjs.com/) who have supported this project from day one.
```

Comment:
Maybe include the contributors who aren't affiliated with either Stripe or Redwood?

Reply:
I'm looking into tools that automatically display contributors' mugshots